packages/desktop-client/src/components/reports/graphs/NetWorthGraph.tsx

```diff
@@ -21,8 +21,14 @@ type NetWorthGraphProps = {
   style?: CSSProperties;
   graphData;
   compact: boolean;
+  domainMinMax;
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏could you please add a type for the new prop?